http/src/main/resources/reference.conf

```diff
@@ -86,6 +86,7 @@ dataproc {
 
 gce {
   customGceImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-gce-debian9-5e2206c"
+  customGceSnapshot = "projects/broad-dsp-gcr-public/global/snapshots/custom-leo-gce-cos-snapshot-d1a330f"
```

Comment:
FYI @Qi77Qi that the snapshot referenced above is the first Jenkins-created one. I've verified it has what we want for the stack downstream :)

Reply:
I could see the benefit but given how easy it is to see the creation time on GCP and how much time I've already spent on this ticket, I'm inclined to not add that change unless you're feeling strongly.

![image](https://user-images.githubusercontent.com/5438223/117717498-e20c2180-b1a8-11eb-9699-a9819b7d0877.png)

Reply:
I think the benefit is if someone see this in config, they're probably not gonna open console and check creation time, vs if the creation time is there in the name, then we're more likely to notice if the image is super old.

Anyways, up to you...